tests/unit/test_config.py

```diff
@@ -49,3 +49,8 @@ def test_default_container_engine(test_dir):
     """The container engine value should default to 'undefined'"""
     config = Config(f"{test_dir}/pipeline.yaml", f"{test_dir}/input_data.yaml", None)
     assert config.container_engine == "undefined"
+
+
+def test_broken_input_files(test_dir):
+    with pytest.raises(RuntimeError):
```

Comment:
I'd strongly recommend providing the `message` that you want to match to.

`with pytest.raises(RuntimeError, message="missing required column(s)")`

Reply:
curse you for making me look up how to do regex for the billionth time

Reply:
done